p2p/service.go

```diff
@@ -0,0 +1,70 @@
+package p2p
+
+import (
+	"context"
+
+	"github.com/pastelnetwork/gonode/common/log"
+	"github.com/pastelnetwork/gonode/p2p/kademlia"
+)
+
+const (
+	logPrefix = "p2p"
+)
+
+// Service represents the p2p service.
+type Service struct {
+	dht DHT
+}
+
+// DHT represents the methods by which a library consumer interacts with a DHT.
+type DHT interface {
+	Store(ctx context.Context, data []byte, key []byte) (id string, err error)
+	Get(ctx context.Context, key string) (data []byte, found bool, err error)
+	CreateSocket() error
+	Listen(ctx context.Context) error
+	GetNetworkAddr() string
+	Bootstrap(ctx context.Context) error
+	UseStun() bool
+	Disconnect() error
+}
```

Comment:
```suggestion
type Client interface {
	Store(ctx context.Context, data []byte, key []byte) (id string, err error)
	Get(ctx context.Context, key string) (data []byte, found bool, err error)
}
```
Please leave only methods that is needed for the `Task`, everything else is for internal use.  Or, if the purpose of it is to be able to replace the Kademlia package in the future, then add another `Client` interface that `NewService` will return.

Reply:
i decided to remove it for now.